app/views/questions/index.html.erb

```diff
@@ -0,0 +1,54 @@
+<%= render :partial => "sidebar/user" %>
+<% if params[:action] == 'popular' %>
+<h2>Popular Questions <small class="hidden-sm">Most viewed questions</small></h2>
+<% elsif params[:action] == 'liked' %>
+<h2>Liked Questions <small class="hidden-sm">Most liked questions</small></h2>
+<% else %>
+<h2>All Questions <small class="hidden-sm">Listing all recent questions</small></h2>
+<% end %>
+<a rel="tooltip" title="Ask a question to the community" data-placement="bottom" href="/post?tags=question:question&template=question&redirect=question" class="btn btn-primary btn-lg pull-right"><i class="fa fa-question-circle fa-white"></i> Ask a Question</a>
```

Comment:
The position of the Ask a question button is a little conjusted at this moment as i just added a placeholder for the button. Will be adjusted with the autocomplete feature suggested in #554